middlewares/pictures.js

```diff
@@ -0,0 +1,38 @@
+const multer = require("multer");
+const setFileSize = require("../utils/setFileSize");
+
+const picturesMiddleware = (
+  postKey = "picture",
+  uploadDir = "tmp/",
+  fileSizeLimit = setFileSize(5, "MB")
+) => {
+  const storage = multer.diskStorage({
+    destination: uploadDir,
+    filename: (req, file, cb) => cb(null, file.originalname),
+    limits: { fileSize: fileSizeLimit },
+  });
+
+  const mimetypeAllowedList = [
+    "image/bmp",
+    "image/gif",
+    "image/jpg",
+    "image/jpeg",
+    "image/png",
+    "image/tiff",
+  ];
+
+  const multerInstance = multer({
+    storage,
+    fileFilter: (req, file, cb) => {
+      const mimetype = file.mimetype;
+      if (!mimetypeAllowedList.includes(mimetype)) {
+        return cb(null, false);
+      }
+      return cb(null, true);
+    },
```

Comment:
tu masz fajnego kandydata na zmiane myslenia o tym, jak czasem mozesz piszac kod, gdy pracujesz z Booleans :)
```js
      const mimetype = file.mimetype;
      const includesMimetype = mimetypeAllowedList.includes(mimetype);
      return cb(null, includesMimetype);
```
byc moze bedziesz chcial zmienic `includesMimetype` na cos, co ma wiecej sensu, ale pomysl skraca Ci sporo kodu, i jesli nazwiesz te stala odpowiednio (tak, zeby kazdy rozumial, ze to jest Bool).

Reply:
odrobina ludzkiego piekna lezy tutaj w tym, ze mozna latwo zapytac "[it] `includesMimetype`?" -- "Yes? No?"
I dosc latwo dochodzimy do Prawda/Nieprawda :)